src/OpenConext/Value/Saml/Metadata/ShibbolethMetadataScope.php

```diff
@@ -50,7 +50,7 @@ public function __construct($scope, $isRegexp = false)
         Assertion::boolean($isRegexp);
 
         if ($isRegexp) {
-            Assertion::validRegularExpression($scope, 'scope');
+            Assertion::validRegularExpression('#' . $scope . '#i', 'scope');
```

Comment:
I have run some tests, escaping non-alphanumeric, non-backslash, non-whitespace characters will give the same result regardless of your delimiter (except if you use a special character as delimiter). So it does not cause any issues when people do escape their slashes.